lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -417,8 +417,8 @@ sub run_backup {
         die $request->{_locale}->text('Error creating backup file');
 
     if ($request->{backup_type} eq 'email') {
-        # suppress warning of single usage of $LedgerSMB::Sysconfig::...
-        no warnings 'once';
```

Comment:
Well, all I know is that before I restructured Sysconfig to the current structure, we had lots of "used only once" errors in the code base. But I see your example does indeed not produce the warning. Nor do my attempts to prove I can generate one.